### PR TITLE
Add loop tab bar example.

### PR DESCRIPTION
**Pull request by sgamerw**
It's a loop tab bar, and the UI window can do in&out anim with the tab bar move direction.

### CODE REVIEW

**Comment by setchi** [comment]
Nice work!

File: Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Example06.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        private Window _currentWindow;
+        
+        void Start()
+        {
+            scrollView.OnSelectionChanged(OnSelectionChanged);
+
+            var items = Enumerable.Range(0, 3)
+                .Select(i => new ItemData($"Tab {i}"))
+                .ToList();
```
Comment (setchi) [inline Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Example06.cs]: @sgamerw
[MUST]
```cs
var items = Enumerable.Range(0, windows.Length)
    .Select(i => new ItemData($"Tab {i}"))
    .ToList();
```
or
```cs
var items = windows
    .Select((_, i) => new ItemData($"Tab {i}"))
    .ToList();
```

File: Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Window.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        [SerializeField] private SlideScreenTransition inAnim = default;
+        [SerializeField] private SlideScreenTransition outAnim = default;
+
+        public void OpenWindow(Scroller.MovementDirection direction)
+        {
+            if (inAnim != null)
+            {
+                inAnim.Direction = direction;
+                inAnim.Animate();
```
Comment (setchi) [inline Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Window.cs]: [nits]
```cs
inAnim.Animate(direction);
```

File: Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Example06.cs (hunk 1 above)
Comment (sgamerw) [reply Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Example06.cs]: Thanks, I will push another request one hour later when I go to work 🙂

**Comment by sgamerw** [comment]
Maybe you should update ReadMe by yourself, I don't know Japanese🙂

**Comment by setchi** [comment]
Thanks!!

I will unify the code style at the Example06 branch.
Being unified is more important than correctness.

I will update the README too!

**Pull request merged**